gradle/java.gradle

```diff
@@ -181,6 +181,7 @@ javadoc {
     setAuthor true
 
     links "https://docs.oracle.com/javase/8/docs/api/"
+    source = 8
```

Comment:
Without this javadoc fails on java 11 due to https://bugs.openjdk.java.net/browse/JDK-8212233